tests/helpers/graphical_units.py

```diff
@@ -0,0 +1,50 @@
+import os
```

Comment:
Comment: it's unclear to me what's the difference between tests/helpers and tests/utils.

Reply:
`helpers `is meant to be used to set new tests up (i.e, generate control data as prerendered .npy, or pre-rendered videos hashes/config), when utils is used by pytest. 

Maybe the wording is not good. Do you have a suggestion?

Reply:
Oh ok. I'd suggest adding a module-level docstring to these two files that specify which one is for the user/dev and which one is for pytest.

On top of that, I'd suggest changing their names to something like dev_tools and pytest_tools or something to that effect. But this may be overkill :shrug:

Reply:
> Oh ok. I'd suggest adding a module-level docstring to these two files that specify which one is for the user/dev and which one is for pytest.

How . By creating a README ?

Reply:
No, by literally adding a docstring at the top of each file.

Reply:
Oh, of course. Will do that